Content.Client/MassMedia/Ui/ArticleEditorPanel.xaml

```diff
@@ -45,6 +45,8 @@
                 <Button Name="ButtonCancel" SetHeight="32" SetWidth="85"
                         StyleClasses="ButtonColorRed" Text="{Loc news-write-ui-cancel-text}"/>
             </Control>
+            <Control HorizontalExpand="True"/>
+              <Label Name="CharCount"  Margin="17 10 0 9" VerticalAlignment="Center"/>
             <Control HorizontalExpand="True"/>
```

Comment:
Both these controls are unnecessary.

Reply:
> Both these controls are unnecessary.

Both?

Reply:
> Both these controls are unnecessary.

One of the controls is the main addition of the PR

Reply:
Or are you talking about the Horizontal Exapnds?

Reply:
> Or are you talking about the Horizontal Exapnds?

As far as I can tell they are

Reply:
@metalgearsloth the controlls are spacers and are necessary to keep the spacing responsive (it emulates the behavior of a flexbox set to space-between in css).
You don't do that kind of spacing with just margins.

Reply:
> @metalgearsloth the controls are spacers and are necessary to keep the spacing responsive (it emulates the behavior of a flexbox set to space-between in css). You don't do that kind of spacing with just margins.

What's wrong with horizontal center alignment + expand? It's just emulating it anyway but with 2 dummy controls.

Reply:
That works I think. 
It might get a bit wonky with centering the text. I've had that happen with the PDA UI.

Reply:
@Brandon-Huu Sloth wants you to remove both `<Control HorizontalExpand="True"/>` and add `HorizontalExpand="True"` to the Label

Reply:
> @Brandon-Huu Sloth wants you to remove both `<Control HorizontalExpand="True"/>` and add `HorizontalExpand="True"` and `HorizontalAlignment="Center"` to the Label

Thank you, I know.

I was gonna do it three days ago and got distracted